landing_page/app/controllers/microposts_controller.rb

```diff
@@ -3,6 +3,9 @@ class MicropostsController < ApplicationController
     before_action :set_micropost, only: [:show, :edit, :update, :destroy, :recover]
 
     include SessionsHelper
+    require 'rake'
+
+    load './lib/tasks/archive_posts.rake'
```

Comment:
Why do you need this here?

Reply:
Addressed in commit 332f81355b6413841644accf64a4944a80ab3eb7